contracts/InterchainTokenService.sol

```diff
@@ -528,7 +497,14 @@ contract InterchainTokenService is
         bytes memory destinationAddress,
```

Comment:
update comment above to indicate only callable by token

Reply:
done